clients/admin-ui/src/features/common/custom-reports/CustomReportTemplates.tsx

```diff
@@ -223,10 +220,6 @@ export const CustomReportTemplates = ({
   const applyDisabled =
     (!fetchedReport && !savedReportId) || fetchedReport?.id === savedReportId;
 
-  if (!userCanSeeReports) {
-    return null;
-  }
-
```

Comment:
I moved this logic to the parent component so none of this loads if the user doesn't have permissions to view it.